scripts/dev/lint_helm_chart.sh

```diff
@@ -16,6 +16,11 @@ fi
 # the binaries yamale and yamllint required by ct are available at `${PROJECT_DIR}/venv/bin`
 export PATH=${PROJECT_DIR}/venv/bin:${PATH}
 
+helm template "${PROJECT_DIR}/helm_chart/" \
+  -f "${PROJECT_DIR}/helm_chart/values.yaml" \
+  -f "${PROJECT_DIR}/helm_chart/values-openshift.yaml" \
+  -f "${PROJECT_DIR}/helm_chart/values-multi-cluster.yaml"
```

Comment:
This doesn't look right: `values-openshift.yaml` and `values-multi-cluster.yaml` are not supposed to be used together, I believe.

Reply:
This line is not necessarily required and I added it just to debug and see if multiple values files are merged and in the failure scenario the eventual `.Values.operator` doesn't have `vaultSecretBackend`. Because that's what seems to be causing our problem. Having all these values files together just increases our chance of catching the values.yaml files merge issue, if there are any.